docs/UserGuide.md

```diff
@@ -158,9 +157,7 @@ Step 5 : Start using the app
 <a name="help"></a>
 ### Viewing help : `help`
 
-If you have trouble using `npc_track`, simply type the `help` command.
-
-Opens the user guide in your browser.
+If you have trouble using `npc_track`, simply type the `help` command. A link to this very user guide will be shown.
```

Comment:
This is not true. The link to the user guide will not be shown, and it will be opened immediately.